README.md

```diff
@@ -106,6 +106,8 @@ After this, we can push the image to the DockerHub image registry:
 $ docker push docker.io/reanahub/reana-demo-agc-cms-ttbar-coffea
 ```
 
+For this example we will not need to build the image but use the image already available in Dockerhub: `docker.io/reanahub/reana-demo-agc-cms-ttbar-coffea:1.0.0` or `registry.cern.ch/docker.io/reanahub/reana-demo-agc-cms-ttbar-coffea:1.0.0` in CERN Harbor registry pull-through cache (this avoids any DockerHub pull limits).
```

Comment:
The CERN Harbor registry is restricted only to CERN people; we could warn about that.